cypress/support/fragments/users/userEdit.js

```diff
@@ -258,4 +265,49 @@ export default {
   selectSingleSelectValue: ({ data }) => {
     cy.do(Select({ label: data.fieldLabel }).choose(data.firstLabel));
   },
+
+  verifyUserPermissionsAccordion() {
+    cy.expect(permissionsAccordion.exists());
+    cy.expect(permissionsAccordion.has({ open: false }));
+  },
+
+  permissionsCount() {
+    permissionsList.perform((el) => {
+      el.invoke('attr', 'aria-rowcount').then((rowCount) => {
+        totalRows = rowCount;
+      });
+    });
+  },
+
+  openSelectPermissions() {
+    cy.do(permissionsAccordion.clickHeader());
+    cy.do(addPermissionsButton.click());
```

Comment:
if we have more than one command in a row, better to pass them as an array

Reply:
This method opens a specific page